tests/unit/stage.py

```diff
@@ -48,3 +48,17 @@ def test_wdir_non_default_is_not_ignored(self):
             self.assertEqual(
                 stage._compute_md5(), "2ceba15e87f6848aa756502c1e6d24e9"
             )
+
+
+class TestPathConversion(TestCase):
+    def test(self):
```

Comment:
I would also add and integration test for this - dvc run. Also, it would be great to test what happens if we have something like dvc run --wdir <win-path> on Windows.

Reply:
@shcheklein There is no need in this case. This unit test is enough. It tests precisely what it should.

Reply:
I'm creating another issue just not to forget this one, @shcheklein : https://github.com/iterative/dvc/issues/1730